### PR TITLE
Added new file mfREADME.md for class Day 1

### PR DESCRIPTION
**Pull request by mfunk**
'cause _teacher_ said so. :smile: 

@Dbarnes1 
@dfoll 
@topowright 
@kgonzago

### CODE REVIEW

**Comment by mfunk** [comment]
related stuff here: #19

File: mfREADME.md [hunk 1]
```diff
@@ -0,0 +1 @@
+*Important stuff goes here**
```
Comment (topowright-zz) [inline mfREADME.md]: What is so important about this?

Comment (Dbarnes1) [inline mfREADME.md]: Looks important enough to me.

Comment (dfoll) [inline mfREADME.md]: @mfunk define "stuff"

**Comment by mfunk** [comment]
**ALL** the important stuff goes here.... so nothing.

**Pull request merged**